lib/ui/window/platform_configuration.cc

```diff
@@ -396,4 +427,22 @@ std::string PlatformConfigurationNativeApi::DefaultRouteName() {
       ->DefaultRouteName();
 }
 
+int64_t PlatformConfigurationNativeApi::GetRootIsolateId() {
+  UIDartState* dart_state = UIDartState::Current();
+  FML_DCHECK(dart_state);
+  return dart_state->GetRootIsolateId();
+}
+
+void PlatformConfigurationNativeApi::RegisterBackgroundIsolate(
+    int64_t isolate_id) {
+  UIDartState* dart_state = UIDartState::Current();
+  FML_DCHECK(dart_state && !dart_state->IsRootIsolate());
+  auto weak_platform_message_handler =
+      (*static_cast<std::shared_ptr<PlatformMessageHandlerStorage>*>(
+           Dart_CurrentIsolateGroupData()))
+          ->GetPlatformMessageHandler(isolate_id);
+  FML_DCHECK(weak_platform_message_handler.lock());
```

Comment:
I think this is probably too strong, someone may write a test that exercises things during shutdown and find this flakily fires on CI.

Reply:
Yea, you are probably right, removed.